### PR TITLE
Merging next-dev branch to main

### PR DESCRIPTION
**Pull request by enricosebastian**
This one is the new Next.js version of the website redone from the ground-up

### CODE REVIEW

**Comment by enricosebastian** [comment]
do you wanna next.js and suffer in react more efficiently? cause you can if you approve this

merge conflicts are small issues

**Review by enricosebastian — COMMENTED** [review]
lgtm yawa (not) mvc ftw

File: package.json [hunk 1]
```diff
@@ -1,21 +1,19 @@
 {
-  "name": "tls_webremaster",
+  "name": "tls_web_remaster",
```
Comment (enricosebastian) [inline package.json:2]: well that's weird lmao

File: components/BannerArticle.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import Link from "next/link";
+
+export default function BannerArticle({article}) {
+    return (
+        <Link href={`/presents/${article.slug}`}>
+            <p dangerouslySetInnerHTML={{__html: article.title.rendered}}/>
```
Comment (rafgabarceo) [inline components/BannerArticle.js:6]: I am not familiar with React's dangerouslySetInnerHTML so I looked it up. According to the following articles:

1. https://stackoverflow.com/questions/29044518/safe-alternative-to-dangerouslysetinnerhtml
2. https://blog.logrocket.com/using-dangerouslysetinnerhtml-in-a-react-application/

It seems that it makes the code vulnerable toward XSS. Will this really be safe? Or will you sanitize the DOM manipulation? What's your reasoning for using it?

File: components/BannerArticles.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import BannerArticle from './BannerArticle';
```
Comment (rafgabarceo) [inline components/BannerArticles.js:1]: Maybe you can update the name of the file to BannerGroup? So that it's easier to read lang naman.

**Review by rafgabarceo — CHANGES_REQUESTED** [review]
Some comments lang on the use of dangerouslySetInnerHTML and some suggestions toward file names.

File: components/BannerArticles.js (hunk 3 above)
Comment (enricosebastian) [reply components/BannerArticles.js:1]: n o

File: components/BannerArticle.js (hunk 2 above)
Comment (enricosebastian) [reply components/BannerArticle.js:6]: the only person that will make this vulnerable is the dumbo webstaffer that injects XSS into the wordpress article lmao

**Pull request merged**